### PR TITLE
feature/cd_flow

### PR DESCRIPTION
**Pull request by walzph**
same as https://github.com/ghga-de/sandbox-storage/pull/15

### CODE REVIEW

**Review by ac-jorellanaf — CHANGES_REQUESTED** [review]
Can you merge the dev branch to this and resolve the conflicts? Thanks.

**Comment by KerstenBreuer** [comment]
Hi @ac-jorellanaf, I rebased the branch to the current dev. Should be fine now. Can you check?

File: Dockerfile [hunk 1]
```diff
@@ -1,29 +1,31 @@
-# Copyright 2021 Universität Tübingen, DKFZ and EMBL for the German Human Genome-Phenome Archive (GHGA)
+# Copyright 2021 Universität Tübingen, DKFZ and EMBL
+# for the German Human Genome-Phenome Archive (GHGA)
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#     https://www.apache.org/licenses/LICENSE-2.0
+#     http://www.apache.org/licenses/LICENSE-2.0
```
Comment (ac-jorellanaf) [inline Dockerfile:8]: Did you also change this?

**Review by ac-jorellanaf — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (KerstenBreuer) [reply Dockerfile:8]: Hm, yes I changed the Dockerfile. In the Template I have it says `http`, but `https` is working, too. Shall we keep it like this to be consistent with the microservice_template_repository?

Comment (ac-jorellanaf) [reply Dockerfile:8]: It should be https at this stage of the WWW, but it seems Apache keeps their license info to use http, so let's keep it as such.

**Pull request merged**